src/pages/api/outboundMatching.ts

```diff
@@ -0,0 +1,111 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import { redirectTo, redirectToError, getUuidFromCookie, setCookieOnResponseObject, getDomain } from './apiUtils';
+import { putStringInS3, UserFareStages } from '../../data/s3';
+import { isCookiesUUIDMatch, isSessionValid } from './service/validator';
+import { MATCHING_COOKIE, USER_DATA_BUCKET_NAME } from '../../constants';
+import { MatchingFareZones } from '../../interfaces/matchingInterface';
+import { Stop } from '../../data/auroradb';
+
+export const putMatchingDataInS3 = async (data: MatchingFareZones, uuid: string): Promise<void> => {
```

Comment:
Maybe we could make this `putOutboundMatchingDataInS3` to distinguish it from the final matching data